ts-rs/src/lib.rs

```diff
@@ -863,15 +863,15 @@ impl<K: TS, V: TS, H> TS for HashMap<K, V, H> {
     type WithoutGenerics = HashMap<Dummy, Dummy>;
 
     fn ident() -> String {
-        "Record".to_owned()
+        "{ [key: any]: any }".to_owned()
```

Comment:
I actually have no idea what this method should return. Maybe it should `panic!`?